atomic_reactor/plugins/pre_koji.py

```diff
@@ -57,6 +62,27 @@ def run(self):
             'enabled': 1,
             'gpgcheck': 0,
         }
+        # If the repo starts with https we should fetch the certificate and update trusts
+        try:
+            if baseurl.startswith("https://"):
+                capath = None
+                if self.koji_ssl_certs:
+                    self.log.info("Using koji certificates from %s" % self.koji_ssl_certs)
+                    capath = os.path.join(self.koji_ssl_certs, 'ca')
+                    with open(capath, 'rb') as cacert_fd:
+                        repo['sslcacert'] = cacert_fd.read()
+                else:
+                    self.log.info("Fetching certificate from %s" % self.pathinfo.topdir)
+                    o = urlparse(self.pathinfo.topdir)
+                    cert = ssl.get_server_certificate((o.hostname, 443))
+                    certpath = '/etc/pki/ca-trust/source/anchors/{}.cert'.format(o.hostname)
+                    self.workflow.files[certpath] = cert
+                    repo['sslcacert'] = certpath
+        except Exception:
+            self.log.debug(traceback.format_exc())
```

Comment:
That's expected - we don't care about ```sslcerify``` if koji gives a plan HTTP link. Although for testing purpose I'll play with it in the next commit